support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -25,6 +25,8 @@ export const pageUrlRegexes = {
 			paperLandingWithGuestCheckout:
 				/\/subscribe\/paper(\/delivery|\/checkout|\/checkout\/guest)?(\?.*)?$/,
 			paperLandingPage: /^\/uk\/subscribe\/paper?(\?.*)?$/,
+			weeklyLandingPage:
+				/\/subscribe\/weekly(\/checkout|\/checkout\/gift)?(\?.*)?$/,
```

Comment:
Should we include `/gift` here (yet)? I presume we'd be happy to push out the GW non-gift test before we've done the work for gifting.

Reply:
Yes , have removed gifting from here , it would skew any results for sure. Updated.